src/main/scala/no/ndla/articleapi/service/converters/HTMLCleaner.scala

```diff
@@ -29,15 +29,16 @@ trait HTMLCleaner {
       moveImagesOutOfPTags(element)
       removeComments(element)
       removeNbsp(element)
-      removeEmptyTags(element)
+      (1 to 3).foreach(_ => removeEmptyTags(element))
```

Comment:
Skjønner. Synes imidlertid det fortjener en kommentar, i praksis det du skrev kondensert ned til én linje... F.eks. `Jsoup doesn't support removing more than one tag at a time. Thus executes the routine 3 times in order to be sure to remove all tags`